tree/tree/inc/TLeafD32.h

```diff
@@ -52,7 +52,7 @@ class TLeafD32 : public TLeaf {
    void            ReadValue(std::istream &s, Char_t delim = ' ') override;
    void            SetAddress(void *add = nullptr) override;
 
-   ClassDefOverride(TLeafD32, 1); // A TLeaf for a 24 bit truncated floating point data type.
+   ClassDefOverride(TLeafD32, 2); // A TLeaf for a 24 bit truncated floating point data type.
```

Comment:
Do we still need the version increases?

Reply:
I guess so. If you look at the Streamer function, I am using there an `if R_v ...`